device/imt/x86_64-im_n29xx_t40n-r0/default_sku

```diff
@@ -0,0 +1 @@
+im_t2624_40x t1
```

Comment:
is it the name still? @AndriiKit 
I think it should be as im_n29xx_t40n-r0, no?

Reply:
Other customers use here device id instead of the platform name.

Reply:
I think here it is up to us. This name is not depend on any name in bfsde.

Reply:
ok